src/test/java/com/google/api/codegen/testdata/java_mock_service_impl_library.baseline

```diff
@@ -297,8 +284,7 @@ public class MockLibraryServiceImpl implements LibraryService  {
   public void updateBookIndex(UpdateBookIndexRequest request,
       StreamObserver<Empty> responseObserver) {
     if (responses != null && !responses.isEmpty()) {
-      Empty response = (Empty) responses.get(0);
-      responses.remove(0);
+      Empty response = (Empty) responses.poll();
```

Comment:
Is it appropriate to pull a response for a method that does not have a response, like this one?

Also, seems better to use `remove()` for the cases that should have a response, which throws a `NoSuchElementException` if there's nothing in `responses`. That's a lot clearer when it's broken than ending up with a `null` in `responseObserver`.